plan-for-extension-exercise/src/planForExtensionExercise/TotalAreaCalculator.java

```diff
@@ -0,0 +1,25 @@
+package planForExtensionExercise;
+
+public class TotalAreaCalculator extends AreaCalculator {
```

Comment:
The TotalAreaCalculator class could get all the functionality of AreaCalculator without extending AreaCalculator (it could just reference the AreaCalculator.calculateArea method) Can you explain why you chose to make the TotalAreaCalculator class a subclass of the AreaCalculator class?